utils/comment.go

```diff
@@ -225,52 +262,30 @@ func generateReviewComment(commentType ReviewCommentType, location formats.Locat
 
 }
 
-func generateApplicabilityReviewContent(issue formats.Evidence, relatedCve formats.CveRow, relatedVulnerability formats.VulnerabilityOrViolationRow, writer outputwriter.OutputWriter) string {
-	remediation := ""
-	if relatedVulnerability.JfrogResearchInformation != nil {
-		remediation = relatedVulnerability.JfrogResearchInformation.Remediation
+func generateApplicabilityReviewContent(issue issues.ApplicableEvidences, writer outputwriter.OutputWriter) string {
+	return outputwriter.GenerateReviewCommentContent(outputwriter.ApplicableCveReviewContent(issue, writer), writer)
+}
+
+func generateSourceCodeVulnerabilityReviewContent(commentType ReviewCommentType, issue formats.SourceCodeRow, writer outputwriter.OutputWriter) (content string) {
```

Comment:
We can unify generateSourceCodeVulnerabilityReviewContent and generateSourceCodeViolationReviewContent by adding a single argument: isViolationContext bool
we can then pass this to the unified function and pass the value to 'violation' arg in IacReviewContent, SastReviewContent, SecretReviewContent